_data/code/municipal/city-of-sault-ste-marie-on.yml

```diff
@@ -0,0 +1,34 @@
+---
+schemaVersion: '1.0'
+adminCode: city-of-sault-ste-marie-on
+releases:
+  - contact:
+      email: d.gowans@cityssm.on.ca
+      name: Dan Gowans
+    date:
+      created: '2017-12-06'
+      metadataLastUpdated: '2020-01-07'
+    description:
+      whatItDoes:
+        en: Parse the MPAC year end tax file.
+        fr: Analyser le fichier d'impôt de fin d'année de la SEFM.
+    category: '43232300'
+    name:
+      en: mpacParser
+      fr: mpacParser
+    licences:
+      - URL:
+          en: 'https://github.com/cityssm/mpacParser/blob/master/LICENSE'
+          fr: 'https://github.com/cityssm/mpacParser/blob/master/LICENSE'
+        spdxID: MIT
+    repositoryURL:
+      en: 'https://github.com/cityssm/mpacParser'
+      fr: 'https://github.com/cityssm/mpacParser'
+    tags:
+      en:
+        - mpac property tax
```

Comment:
Are these supposed to be separate entries in a list rather than a single string? Maybe a bug in the form

Reply:
That would be me.  I thought the tags were supposed to be separated by spaces.

Reply:
No problem, I think the current setup is that you click the "Add more tags" button to add multiple values.
I'll land this as-is and it can be edited after :)

Reply:
I did see that button, but I said, "I can fit my tags in this box."  : )